### PR TITLE
Apply some fixes from sharkey300#9117

### PR DESCRIPTION
**Pull request by TymanWasTaken**
# Full changelog:

### Added:
- Game Annihilator wiki url
- Parented the gift colors

### Changed:
- Fixed turbo potato 5 requirement in lore
- Changed daedalus stats to base stats
- Made spooky pie & crab hat details obfuscated
- Changed growth armor stats to base stats
- Fixed hunk of blue ice lore language
- Made Ornate Zombie Sword not a dungeon item

### CODE REVIEW

**Comment by TymanWasTaken** [comment]
This is done but someone fix this https://canary.discord.com/channels/516977525906341928/693600396492537896/810990355536412722 who knows how to

File: items/DAEDALUS_AXE.json [hunk 1]
```diff
@@ -4,10 +4,7 @@
   "nbttag": "{Unbreakable:1b,HideFlags:254,display:{Lore:[0:\"§7Damage: §c+124\",1:\"\",2:\"§7Health: §a+43 HP\",3:\"§7Intelligence: §a+43\",4:\"\",5:\"§7Gains §c+4 Damage §7per Taming level\",6:\"§7Copies the stats from your active pet\",7:\"\",8:\"§7Earn §6+35 coins §7from monster kills\",9:\"\",10:\"§7§8This item can be reforged!\",11:\"§6§lLEGENDARY SWORD\"],Name:\"§6Daedalus Axe\"},ExtraAttributes:{id:\"DAEDALUS_AXE\"}}",
```
Comment (IRONM00N) [inline items/DAEDALUS_AXE.json:4]: fix the nbt tag too

File: items/SPOOKY_PIE.json [hunk 2]
```diff
@@ -8,9 +8,9 @@
     "§6Spooky Festival §7during",
     "§7SkyBlock Year 72.",
     "",
-    "§7Player: §6IRONM00N",
-    "§7Position: §a#4,353",
-    "§7Candy Score: §a651",
+    "§7Player: §6§kIRONM00N",
```
Comment (IRONM00N) [inline items/SPOOKY_PIE.json:11]: 😠This is my easter egg, Moulberry said I could have it

Comment (TymanWasTaken) [reply items/SPOOKY_PIE.json:11]: Well your name is still there and the other items were formatted like this so

File: items/SPOOKY_PIE.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
     "§6Spooky Festival §7during",
     "§7SkyBlock Year 72.",
     "",
-    "§7Player: §6§kIRONM00N",
+    "§7Player: §6IRONM00N",
```
Comment (TymanWasTaken) [inline items/SPOOKY_PIE.json:11]: bruh at least make it consistant

**Pull request merged**